docs/Login.md

```diff
@@ -126,7 +126,7 @@ You should see
 
 ---
 
->**Creating a Login Page**
+>**Mojolicious Session Tutorial - Page 1**
```

Comment:
This is in the title bar of the browser.  The message passed is correct.  I can see why you thought it was a bug.